src/application-navigation/terra-application-navigation/tabs/_TabCount.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+import { useAnimatedCount } from '../utils/helpers';
+
+import styles from './TabCount.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The number of notifications to display.
+   */
+  value: PropTypes.number,
+  /**
+   * Whether or not the count represents rolled up counts.
+   */
+  isRollup: PropTypes.bool,
+  /**
+   * @private
+   * Object containing intl APIs.
+   */
+  intl: intlShape,
+};
+
+const TabCount = ({ value, isRollup, intl }) => {
+  const countRef = useRef();
+
+  useAnimatedCount(countRef, value);
+
+  let validatedValue = value;
+  if (isRollup) {
+    validatedValue = intl.formatMessage({ id: 'Terra.applicationNavigation.notifications.new' });
+  } else if (value >= 999) {
+    validatedValue = '99+';
```

Comment:
Should this be 999+ like https://github.com/cerner/terra-application/pull/87/files#diff-31daef00e0f7bfe3c5f4c624e2a8575fR24?

Reply:
Hmm this definitely feels off, but I'm not sure whether we should update the check or update the string. @dkasper-was-taken do you remember?

Reply:
I think the if check portion is likely what is off. A navigation tab has a width restriction for fitting 99+. I feel like we fixed this elsewhere, but may have missed this one.

Reply:
I updated this, and fixing in terra-framework with https://github.com/cerner/terra-framework/pull/1259